CHANGELOG.md

```diff
@@ -557,7 +561,9 @@ searchField
 ## [0.2.2] - 2019-03-29
 
 - First stable release.
-[main]: https://github.com/square/Blueprint/compare/0.24.0...HEAD
+
+[main]: https://github.com/square/Blueprint/compare/0.25.0...HEAD
+[0.24.0]: https://github.com/square/Blueprint/compare/0.24.0...0.25.0
```

Comment:
oh, looks like we missed this last time — we need links for both `0.24.0` and `0.25.0`, if you don't mind